avivator/src/components/RenderingModeSelect.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+
+import { RENDERING_MODES } from '../../../dist'; // eslint-disable-line import/extensions,import/no-unresolved
```

Comment:
nit, is there a way to set an alias for viv in the webpack config?

```javascript
const path = require('path');

module.exports = {
  //...
  resolve: {
    alias: {
      '@hms-dbmi/viv': '../dist',
    },
  },
};
```

And then imports in avivator are:

```javascript
 import { RENDERING_MODES } from '@hms-dbmi/viv';
```

Reply:
Good call!